code/procs/mobprocs/chatprocs.dm

```diff
@@ -939,3 +977,26 @@
 			thisR = "<span class='adminHearing' data-ctx='[M.client.chatOutput.getContextFlags()]'>[thisR]</span>"
 		if(thisR != "")
 			M.show_message(thisR, 2)
+
+// I am deeply moved every time ;_;
+/proc/danny_boy(var/index)
+	var/danny_boy_lyrics = list(
+		"Oh, Danny boy, the pipes, the pipes are calling",
+		"From glen to glen, and down the mountain side.",
```

Comment:
this would probably be better off going in `strings/` and picking via one of our helper procs to do so